drivers/bluetooth/hci/hci_silabs_siwx91x.c

```diff
@@ -32,6 +75,17 @@ static int siwx91x_bt_open(const struct device *dev, bt_hci_recv_t recv)
 	return status ? -EIO : 0;
 }
 
+static int siwx91x_bt_setup(const struct device *dev, const struct bt_hci_setup_params *params)
+{
+    int ret = rsi_bt_driver_send_tx_pwr_vs_cmd(dev);
```

Comment:
Please be consistent with error variable naming. You're using `err` in the other place, and that's also what I'd prefer here. Also, I think this will look a bit cleaner as:
```c
{
	int err;

	err = rsi_bt_driver_send_tx_pwr_vs_cmd(dev);
	if (err < 0) {
		...
```

The main point is that if you ever need to extend this with more commands then it's not very clean to do the first one immediately when you declare of the variable.